apple/cc_toolchain_forwarder.bzl

```diff
@@ -16,8 +16,8 @@
 A rule for handling the cc_toolchains and their constraints for a potential "fat" Mach-O binary.
 """
 
-load("@build_bazel_rules_apple//apple/internal:providers.bzl", "new_appleplatforminfo")
-load("@build_bazel_rules_apple//apple:providers.bzl", "ApplePlatformInfo")
+load("//apple/internal:providers.bzl", "new_appleplatforminfo")
+load("//apple:providers.bzl", "ApplePlatformInfo")
 load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain", "use_cpp_toolchain")
```

Comment:
After doing this, can you reorder the loads? Relative ones should be below external ones.

Reply:
Related: I noticed buildifier isn't run on PR's here. By design or should that be added?

Reply:
It's checked in `buildkite/rules-apple-darwin/pr/buildifier`, right?

Reply:
(Though maybe it's not as strict as it should be.)

Reply:
Ran buildifier on the repo locally/manually and got this diff and others like it. I'll look to see if it's a version diff or something.

![CleanShot 2024-07-11 at 14 43 48@2x](https://github.com/bazelbuild/rules_apple/assets/75236/7f4decc2-bbcd-49e5-9b4c-55dd874a6e0e)

Reply:
I'll make the sorts another PR, and if I see a buildifier fix I'll include that with it.